crates/core/migrations/2022-08-15-172917_add_compound_key_to_candymachine_creators/down.sql

```diff
@@ -0,0 +1,4 @@
+-- This file should undo anything in `up.sql`
+alter table candy_machine_creators
+drop constraint candy_machine_creators_compound_pkey,
+add constraint candy_machine_creators_pkey primary key (candy_machine_creators_compound_pkey);
```

Comment:
Why is `candy_machine_creators_compound_pkey` added as primary key? This should be candy_machine_address if you want to revert it back

Reply:
yeah, that was my intention, oops!